### DISCUSSION TITLE
How to get user info for mentioned users

### DISCUSSION THREAD

**Discussion by MUKAPP** [eternnoir/pyTelegramBotAPI#2039] [Q&A] [answered]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.13.0
2. What OS are you using?
Windows 11
3. What version of python are you using?
3.11

I've noticed that the user object for mentions returns None, so I can only get the mentioned user's username. How can I get the full user info for mentioned users?

**Comment by Badiboy** [comment, marked as answer]
If user object is None and only username is available - thus you cannot.

**Reply by MUKAPP** [reply]
Ahh. Okay, thanks.
